version.go

```diff
@@ -18,4 +18,4 @@ limitations under the License.
 
 package sdk
 
-const Version = "0.1.287"
+const Version = "0.1.288"
```

Comment:
we update version only when we tag so we can remove it  from this PR

Reply:
@pvasant What do you mean by that?
I will reset the version to 0.1.287 then